### PR TITLE
Created Enterprise security and compliance section

### PR DESCRIPTION
**Pull request by kendradello**
Removed sections about WDAG and WIP and replaced it with an Enterprise section

**Rendered article section:**
https://review.docs.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-2088#enterprise-security-and-compliance

### CODE REVIEW

File: microsoft-edge/privacy-whitepaper/index.md [hunk 1]
```diff
@@ -196,6 +196,13 @@ Microsoft Edge lets you download files safely and securely.  To choose where fil
 
 To view the history of your previous downloads, go to `edge://downloads`.  To clear your browsing data and delete your download history, go to `edge://settings/clearBrowserData`.  Deleting your download history from Microsoft Edge does not remove the files from your device.  Deleting downloaded files from your device does not remove the files from your download history.  When you use **InPrivate** browsing or **Guest** mode, the download history from the session is cleared when you close the **InPrivate** or **Guest** windows. The files remain saved on the device.
 
+<!-- ====================================================================== -->
+## Enterprise Security & Compliance
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:200]:
```suggestion
## Enterprise Security and Compliance

```
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/ampersand

File: microsoft-edge/privacy-whitepaper/index.md [hunk 2]
```diff
@@ -196,6 +196,13 @@ Microsoft Edge lets you download files safely and securely.  To choose where fil
...
+Microsoft Edge supports a variety of first party enterprise security & compliance features including, Windows Information Protection, Microsoft Defender Application Guard, Microsoft Endpoint DLP, and Microsoft Insider Risk management. These features become functional only on an Enterprise managed devices either by acquiring the right license for the feature or the administrator deploying specific policies on the device.
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:201]:
```suggestion
Microsoft Edge supports a variety of first party enterprise security and compliance features including, Windows Information Protection, Microsoft Defender Application Guard, Microsoft Endpoint DLP, and Microsoft Insider Risk management. These features become functional only on an Enterprise managed devices either by acquiring the right license for the feature or the administrator deploying specific policies on the device.
```
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/ampersand

File: microsoft-edge/privacy-whitepaper/index.md (hunk 1 above)
Comment (kendradello) [reply microsoft-edge/privacy-whitepaper/index.md:200]: Working with the team to understand if this is a brand name. If not, I will make the '&' update and make change Security and Compliance to lowercase

**Comment by kendradello** [comment]
@mikehoffms how can I verify the docs links I included are valid?

File: microsoft-edge/privacy-whitepaper/index.md [hunk 3]
```diff
@@ -197,8 +197,8 @@ Microsoft Edge lets you download files safely and securely.  To choose where fil
...
+## Enterprise security and compliance
+Microsoft Edge supports a variety of first party enterprise security and compliance features including, [Windows Information Protection](/windows/security/information-protection/windows-information-protection/protect-enterprise-data-using-wip), [Microsoft Defender Application Guard](/windows/security/threat-protection/microsoft-defender-application-guard/md-app-guard-overview), [Microsoft Endpoint Data Loss Prevention (DLP)](/microsoft-365/compliance/endpoint-dlp-learn-about?view=o365-worldwide), and [Microsoft Insider Risk Management](/microsoft-365/compliance/insider-risk-management-solution-overview?view=o365-worldwide). These features become functional only on enterprise managed devices by acquiring the right license for the feature or the administrator deploying specific policies on the device.
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md]:
```suggestion

Microsoft Edge supports a variety of first-party enterprise security and compliance features, including [Windows Information Protection](/windows/security/information-protection/windows-information-protection/protect-enterprise-data-using-wip), [Microsoft Defender Application Guard](/windows/security/threat-protection/microsoft-defender-application-guard/md-app-guard-overview), [Microsoft Endpoint Data Loss Prevention (DLP)](/microsoft-365/compliance/endpoint-dlp-learn-about?view=o365-worldwide), and [Microsoft Insider Risk Management](/microsoft-365/compliance/insider-risk-management-solution-overview?view=o365-worldwide). These features become functional only on enterprise-managed devices by acquiring the right license for the feature or by the administrator deploying specific policies on the device.
```

File: microsoft-edge/privacy-whitepaper/index.md [hunk 4]
```diff
@@ -197,8 +197,8 @@ Microsoft Edge lets you download files safely and securely.  To choose where fil
 To view the history of your previous downloads, go to `edge://downloads`.  To clear your browsing data and delete your download history, go to `edge://settings/clearBrowserData`.  Deleting your download history from Microsoft Edge does not remove the files from your device.  Deleting downloaded files from your device does not remove the files from your download history.  When you use **InPrivate** browsing or **Guest** mode, the download history from the session is cleared when you close the **InPrivate** or **Guest** windows. The files remain saved on the device.
 
 <!-- ====================================================================== -->
-## Enterprise Security & Compliance
-Microsoft Edge supports a variety of first party enterprise security & compliance features including, Windows Information Protection, Microsoft Defender Application Guard, Microsoft Endpoint DLP, and Microsoft Insider Risk management. These features become functional only on an Enterprise managed devices either by acquiring the right license for the feature or the administrator deploying specific policies on the device.
+## Enterprise security and compliance
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:200]:
```suggestion
## Enterprise security and compliance

```
blank line convention

File: microsoft-edge/privacy-whitepaper/index.md [hunk 5]
```diff
@@ -196,6 +196,13 @@ Microsoft Edge lets you download files safely and securely.  To choose where fil
...
 
+<!-- ====================================================================== -->
+## Enterprise security and compliance
+Microsoft Edge supports a variety of first party enterprise security and compliance features including, [Windows Information Protection](/windows/security/information-protection/windows-information-protection/protect-enterprise-data-using-wip), [Microsoft Defender Application Guard](/windows/security/threat-protection/microsoft-defender-application-guard/md-app-guard-overview), [Microsoft Endpoint Data Loss Prevention (DLP)](/microsoft-365/compliance/endpoint-dlp-learn-about), and [Microsoft Insider Risk Management](/microsoft-365/compliance/insider-risk-management-solution-overview). These features become functional only on enterprise managed devices by acquiring the right license for the feature or the administrator deploying specific policies on the device.
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:201]:
```suggestion
Microsoft Edge supports a variety of first-party enterprise security and compliance features, including:
* [Windows Information Protection](/windows/security/information-protection/windows-information-protection/protect-enterprise-data-using-wip)
* [Microsoft Defender Application Guard](/windows/security/threat-protection/microsoft-defender-application-guard/md-app-guard-overview)
* [Microsoft Endpoint Data Loss Prevention (DLP)](/microsoft-365/compliance/endpoint-dlp-learn-about)
* [Microsoft Insider Risk Management](/microsoft-365/compliance/insider-risk-management-solution-overview) 

These features become functional only on enterprise-managed devices by acquiring the right license for the feature or by the administrator deploying specific policies on the device.
```

**Comment by mikehoffms** [comment]
@kendradello 
To test DMC urls where there's no domain prefix in .md source file:

Click the link in the rendered page
https://review.docs.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-2088#enterprise-security-and-compliance
or manually add prefix: docs.microsoft.com
eg open new tab, start typing docs... paste a suffix url after ".com"

**Comment by kendradello** [comment]
Verified all doc links using the instructions. Ready for sign off.

File: microsoft-edge/privacy-whitepaper/index.md [hunk 6]
```diff
@@ -196,6 +196,20 @@ Microsoft Edge lets you download files safely and securely.  To choose where fil
...
+* [Windows Information Protection](/windows/security/information-protection/windows-information-protection/protect-enterprise-data-using-wip)
+* [Microsoft Defender Application Guard](/windows/security/threat-protection/microsoft-defender-application-guard/md-app-guard-overview)
+* [Microsoft Endpoint Data Loss Prevention (DLP)](/microsoft-365/compliance/endpoint-dlp-learn-about)
+* [Microsoft Insider Risk Management](/microsoft-365/compliance/insider-risk-management-solution-overview) 
+
+These features become functional only on enterprise-managed devices by acquiring the right license for the feature or by the administrator deploying specific policies on the device.
+
+When these enterprise features are on, the browser sends auditing and diagnostic data to the administrator as per the feature’s requirement to function, manage or diagnose properly. This data collection is controlled by the enterprise administrator and end user is not able to opt-out.
+
+Please contact your enterprise administrator to understand your company’s policies for such data collection.
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:212]:
```suggestion
Please contact your enterprise administrator to understand your company's policies for such data collection.
```

**Review by mikehoffms — APPROVED** [review]

**Pull request merged**